consensus/polybft/extra_test.go

```diff
@@ -0,0 +1,484 @@
+package polybft
+
+import (
+	"crypto/rand"
+	mrand "math/rand"
+	"testing"
+
+	"github.com/0xPolygon/polygon-edge/chain"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/bitmap"
+	bls "github.com/0xPolygon/polygon-edge/consensus/polybft/signer"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/wallet"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/hashicorp/go-hclog"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/umbracle/fastrlp"
+)
+
+func TestExtra_Encoding(t *testing.T) {
```

Comment:
Test can be parallelized with `t.Parallel()`

Reply:
Fixed through #825